### PR TITLE
Update Tripadvisor

### PR DESCRIPTION
**Pull request by ThomasJClark**
### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Update the Tripadvisor name (lowercase "a"), color, and logomark

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Thanks for your work on this, @ThomasJClark. Colour & source URL look good to me. Your SVG, however, is just a shade over 24px in width.

**Comment by ThomasJClark** [comment]
I think the precision on svgo was a little too high, should be fixed now

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4741,9 +4741,9 @@
             "source": "https://www.trendmicro.com/"
         },
         {
-            "title": "TripAdvisor",
-            "hex": "00AF87",
-            "source": "https://tripadvisor.mediaroom.com/download/TripAdvisor_Logo_Guidelines_5_15_17.pdf"
+            "title": "Tripadvisor",
+            "hex": "34E0A1",
+            "source": "https://filecache.mediaroom.com/mr5mr_tripadvisor/177928/download/Tripadvisor%20Logo%20%26%20Guidelines%201.29.pdf"
```
Comment (phatbhoy67) [inline _data/simple-icons.json]:
```suggestion
            "source": "https://filecache.mediaroom.com/mr5mr_tripadvisor/177928/download/Tripadvisor%20Logo%20%26%20Guidelines%201.29.pdf"
```
```suggestion
            "source": "https://tripadvisor.mediaroom.com/logo-guidelines"
```

**Comment by PeterShaggyNoble** [comment]
SVG looks good to me now, too, thanks for the update :+1:

And I agree that @phatbhoy67's choice for the source URL would be more suitable.

**Comment by ericcornelissen** [comment]
Thanks for updating the Tripadvisor logo @ThomasJClark! Your contribution is much appreciated :smiley:

**Pull request merged**